pages/data-providers/[data-provider-id]/overview.jsx

```diff
@@ -39,7 +62,7 @@ const DataStatisticsCard = ({
   className,
   ...restProps
 }) => (
-  <Card className={className} {...restProps}>
+  <OverviewCard className={className} {...restProps}>
```

Comment:
If you do not append own classes, `className` can be passed implicitly via `restProps`